### ISSUE TITLE
Buttons not clickable in Safari iOS on 0.17.4.rc4

### ISSUE THREAD

**Issue by DylanPKing** [LemmyNet/lemmy-ui#1134]
Found a bug? Please fill out the sections below. 👍

For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy)

### Issue Summary

When browsing on Safari iOS 16.5, buttons are not clickable, including the menu button in the top right, and the Sidebar [+] button, and the Subscribed/Local/All Buttons.

### Steps to Reproduce

1. Visit Lemmy.ml on an iOS device in safari
2. Try to open the menu in the top right.

### Technical details

- Please post your log: `sudo docker-compose logs > lemmy_log.out`.
- What OS are you trying to install lemmy on?
- Any browser console errors?

**Comment by dessalines** [comment]
I don't have any apple devices so I can't test this.

**Comment by DylanPKing** [comment]
I've just tested and I can also repro on Firefox 114.0.1 on windows See the attached videos showing it both with a wide and a narrow window

https://github.com/LemmyNet/lemmy-ui/assets/37106032/b3aa9a92-a5cf-4dea-8452-7ac485f62004

https://github.com/LemmyNet/lemmy-ui/assets/37106032/853b6d06-aa6c-44f2-ab93-622d074087d8

If I can find a way to attach logs from firefox I'll add them here.

**Comment by michaelfarrellj** [comment]
Did you add any links to your profile bio? Lemmy-UI was working for me on everything up until I added my Mastodon link to my bio, at which point the whole UI broke and I have the exact same issue as you: buttons are not clickable, not on my iPhone in Safari and not on my Windows PC in Chrome. I've tried to edit my profile in settings, but the "Save Changes" button is not clickable either. For some reason, I can like and comment while using Mlem, but Mlem doesn't have a way to edit the profile, which I assume is what caused my problem.

**Comment by michaelfarrellj** [comment]
I have found a fix on the user end that made Lemmy-UI usable for me again. Here is what I did: 

1. Ask various friends with Android devices if I could use their device. The closer the friend, the better. If you do this, make it clear that you will be downloading an app and that if they want to watch everything you do that it's perfectly fine.
2. Download Jerboa for Lemmy onto the device from the Google Play Store.
3. Sign in to Lemmy account in Jerboa on the Android device.
4. Go to account settings.
5. Delete profile and header photos, as well as profile bio which contained Mastodon link.
6. Save changes.
7. Open Lemmy on iOS or PC to check if it is now working. I did have to refresh and sign back in on my personal devices.
8. Check to see if everything is working properly.
9. Sign out of Jerboa and delete app from friend's phone.
10. Thank them.

Fortunately Jerboa has a little more functionality than Mlem, and fortunately I had a friend willing to lend me his Android device. Buttons are now clickable for me on all devices. However, it doesn't actually solve the real problem, whatever that was.

**Comment by cakmakb** [comment]
Hey guys,

I just wanted to confirm this issue. I practically can't use lemmy.ml/feddit.de on my iPhone SE 2020 (Safari, Edge, Firefox) and on my Macbook (Safari, Edge). The buttons, hamburger menu etc. seem to work while not logged in. If I log in nearly everything stops working and I can only see tooltips sometimes. I think this bug is related to this issue: #684 

Everything worked great on my Windows Laptop. But all my Apple devices are flawed. I just wanted to emphasize that this issue may exclude a big chunk of a potential users on the first try and they may never come back. 

I did nothing fancy to my accounts. I just subscribed, commented and voted. My account on feddit.de was completely unused but had the same issue.
